Storage/DoctrineStorage.php

```diff
@@ -150,13 +150,9 @@ private function tableExists() {
 	}
 
 	private function createTable() {
-		// TODO remove as soon as Doctrine DBAL >= 3.0 is required
-		$stringType = defined('Doctrine\DBAL\Types\Types::STRING') ? Types::STRING : Type::STRING;
-		$arrayType = defined('Doctrine\DBAL\Types\Types::ARRAY') ? Types::ARRAY : Type::TARRAY;
-
 		$table = new Table(self::TABLE, [
-			new Column($this->keyColumn, Type::getType($stringType)),
-			new Column($this->valueColumn, Type::getType($arrayType)),
+			new Column($this->keyColumn, Type::getType(Types::STRING), ['length' => 50]),
```

Comment:
It is deprecated in doctrine/dbal 3.4 to not set the length in a `string` type column, by default it is 255, for the key only `craue_form_flow` is used, so 50 should be enough.